src/main/scala/com/atomist/rug/function/github/CreateTagFunction.scala

```diff
@@ -35,15 +31,14 @@ class CreateTagFunction
     logger.warn(s"Invoking createTag with tag '$tag', message '$message', sha '$sha', owner '$owner', repo '$repo', apiUrl '$apiUrl' and token '${safeToken(token)}'")
 
     val apiUrlSlash = if (apiUrl.endsWith("/")) apiUrl else s"$apiUrl/"
+    val ghs = gitHubServices(token, apiUrlSlash)
```

Comment:
the method `def gitHubServices(token: String, apiUrl: String): GitHubServices =
    GitHubServices(token, Option(apiUrl))` doesn't add value. I will inline them all